### DISCUSSION TITLE
You want the editor to support merging polygons

### DISCUSSION THREAD

**Discussion by xiaoliang6669** [godotengine/godot-proposals#5795] [Editor]
I hope the editor supports polygon merging, because we don't know the official rules, and the merging of our own code will cause various problems. It is convenient for us to generate polygons from the coordinate obstacle information of maps in other games and import them into GODOT for navigation
![QQ图片20221117110504](https://user-images.githubusercontent.com/112788136/202344920-5e5951fc-f33b-408e-9b71-021bd111f90d.png)
This tool is too weak to recognize many
![999999](https://user-images.githubusercontent.com/112788136/202365072-64387636-31c9-40a3-8c36-251b55a381f4.png)

**Comment by Zireael07** [comment]
Not the place for support questions. Use e.g. the Q&A site or the Godot Forums instead.
